dashboard/Dockerfile

```diff
@@ -13,13 +13,16 @@ RUN gem install compass
 RUN npm install bower grunt-cli -g &&\
   npm install grunt
 
-RUN cd /opt &&\
-  git clone https://github.com/codedellemc/mars-challenge &&\
-  cd /opt/mars-challenge/dashboard/ &&\
-  npm install &&\
-  bower install --allow-root --force-latest --config.interactive=false &&\
-  grunt build &&\
-  cp -r dist /app
+WORKDIR /opt
+RUN git clone https://github.com/codedellemc/mars-challenge
+
+WORKDIR /opt/mars-challenge/dashboard/
+
+RUN npm install
```

Comment:
Much easier to troubleshoot and you don't have to go through the 5 minute process every time if something breaks at the end.